UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Validation/ProductNamesValidatorTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Linq;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+using UKHO.ExchangeSetService.API.Validation.V2;
+using UKHO.ExchangeSetService.Common.Models.V2.Request;
+
+namespace UKHO.ExchangeSetService.API.UnitTests.Validation
```

Comment:
Can we create a subfolder V2 in Validation for Unit Tests as well, to keep it same as in API folder structure

Reply:
Moved to subfolder